### PR TITLE
3185-csv-import: new impoter, propper categories

### PR DESCRIPTION
**Pull request by dux**

### CODE REVIEW

File: app/flow_io/decorators/spree_variant_decorator.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def self.flow_truncate
 
   # syncs product variant with flow
   def flow_sync_product
-    # flow_client.items.put_by_number flow_org, sku, flow_item
+    return unless product.images.first
```
Comment (mbryzek) [inline app/flow_io/decorators/spree_variant_decorator.rb]: why require an image? flow's apis do not require images

File: lib/import/custom_products.rb [hunk 2]
```diff
@@ -26,33 +28,34 @@ def count
     end
 
     def get_row
-      row = @rows.shift
+      row = @rows.shift || return
+      row.map!{ |el| el == ' ' ? nil : el }
```
Comment (mbryzek) [inline lib/import/custom_products.rb]: maybe el.strip.empty?

Comment (dux) [inline lib/import/custom_products.rb]: of course ;)

File: app/flow_io/decorators/spree_variant_decorator.rb (hunk 1 above)
Comment (dux) [inline app/flow_io/decorators/spree_variant_decorator.rb]: this was a fix when bulk inserting products to the database.

all products we have an image, but it is possible that images are uploaded in another thread so they are not available when requested.

prices will be right and catalogs can be re-synced to get fresh images in the console, so I will fix to not request an image for sync.

**Pull request merged**